source/Sashimi.AzureResourceGroup/AzureResourceGroupActionHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Sashimi.AzureScripting;
+using Sashimi.Server.Contracts.ActionHandlers;
+
+namespace Sashimi.AzureResourceGroup
+{
+    class AzureResourceGroupActionHandler : IActionHandlerWithAccount
+    {
+        public string Id => SpecialVariables.Action.AzureResourceGroup.ResourceGroupActionTypeName;
+        public string Name => "Deploy an Azure Resource Manager template";
+        public string Description => "Deploy an Azure Resource Manager (ARM) template.";
+        public string? Keywords => null;
+        public bool ShowInStepTemplatePickerUI => true;
+        public bool WhenInAChildStepRunInTheContextOfTheTargetMachine => false;
+        public bool CanRunOnDeploymentTarget => false;
+        public ActionHandlerCategory[] Categories => new[] { ActionHandlerCategory.BuiltInStep, AzureConstants.AzureActionHandlerCategory };
```

Comment:
Server has 3 categories on the definition `ActionHandlerCategory.BuiltInStep, ActionHandlerCategories.Azure, ActionHandlerCategory.Script`

Reply:
Fixed